### PR TITLE
Replaced header image with real headings

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Replaced an image of headers with real headers.

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/markdown/concepts/headings/headings.md [hunk 1]
```diff
@@ -35,7 +35,15 @@ The number of number signs you use should correspond to the heading level. For e
 ###### Heading level 6
 ```
 
-![example](https://i.imgur.com/8NNe3NE.png)
+<details>
```
Comment (Dusch4593) [inline content/markdown/concepts/headings/headings.md:38]:
```suggestion
<details open>
```

I prefer if the `<details>` were already open upon page load (and then the user can collapse/expand if they want).

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ethan-master-coding Thanks for this edit! I left only one comment/suggestion but I wanted to make one more here. On line # 3, could we shorten the meta-`Description` to 150-characters or less?

**Comment by ethan-master-coding** [comment]
@Dusch4593 accepted the change. And is ok that I broke the no plagiarize rule. I copied it from the HTML heading entry and changed a few things to fit MD heading.

File: content/markdown/concepts/headings/headings.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Headings'
-Description: 'Headings are titles or subtitles that you want to display with Markdown. There is a total of 6 different headings. To create a heading, add hashtag signs # in front of a word or phrase. The number of number signs you use should correspond to the heading level. For example, to create a heading level three , use three number signs (e.g., ### My Header). markdown  example Alternatively, on the line below the text:'
+Description: 'There are 6 heading elements in markdown. They are made with #, with 1 being the largest sized heading and 6 being the smallest. These are great for labeling different sections in a website.'
```
Comment (Dusch4593) [inline content/markdown/concepts/headings/headings.md:3]:
```suggestion
Description: 'Headings are markdown elements used for displaying titles, subtitles, or different sections in a website. There are six headings in total. They are made with a hash symbol, #, with 1 being the largest and 6 being the smallest.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@ethan-master-coding Just this small edit in `Description` and we should be good to merge.

**Comment by ethan-master-coding** [comment]
@Dusch4593 accepted the change. (And its ok I broke that rule right)

**Comment by Dusch4593** [comment]
> @Dusch4593 accepted the change. (And its ok I broke that rule right)

As long as you are not overtly copy/pasting large amounts of content from other sources and not giving due credit, this should be fine. Plus, it seems that you were borrowing a piece of text from another Codecademy Docs article, which seems to be within the bounds.

**Review by Dusch4593 — APPROVED** [review]

**Comment by ethan-master-coding** [comment]
@Dusch4593 It was meant to be more of a joke than a serious thing. And Lyall wants your input on #674

**Comment by Dusch4593** [comment]
@ethan-master-coding Lol This is why I don't think plain text is always the best for conveying things like humor. I'm more than happy to help Lyall with their PR and will try to address their question asap. But fwiw, Lyall should tag me directly in [the issue/PR at hand](https://github.com/Codecademy/docs/issues/674) if they really need to get my attention.

**Comment by ethan-master-coding** [comment]
> Lol This is why I don't think plain text is always the best for conveying things like humor.
Yea should of added some emojis.

> But fwiw, Lyall should tag me directly in https://github.com/Codecademy/docs/issues/674 if they really need to get my attention.

We were talking in dms a week+ ago about this but I forgot to mention you about it.

**Comment by Dusch4593** [comment]
@ethan-master-coding I just noticed something concerning, now that you mention [HTML: Headings](https://www.codecademy.com/resources/docs/html/headings). It looks like the markup part of the content is not rendering correctly (even though its coded correctly as `<h1>` through `<h6>`):

Screen Shot 2022-04-19 at 5 35 28 PM

I believe Chris from Engineering is aware of this issue but has yet to find the reason behind this. Out of caution, could this PR move forward with the image of headings still in place? In the future (which I hope is soon), we should revisit and add markup like `<details>` and `<summary>` if possible.

**Comment by ethan-master-coding** [comment]
@Dusch4593 it may be because we are using HTML instead of what it is in MD. And is it this Chris: https://discuss.codecademy.com/u/jibs/summary

**Comment by Dusch4593** [comment]
> @Dusch4593 it may be because we are using HTML instead of what it is in MD. And is it this Chris: https://discuss.codecademy.com/u/jibs/summary

Really hard to tell from the profile and cover images. Plus, I don't visit the forums as much as I used to.

**Comment by lyallstewart** [comment]
> But fwiw, Lyall should tag me directly in https://github.com/Codecademy/docs/issues/674 if they really need to get my attention.

I did 🙂

**Comment by Dusch4593** [comment]
> > But fwiw, Lyall should tag me directly in #674 if they really need to get my attention.
> 
> I did 🙂

Ah yes, you did! Will stop by to offer my thoughts as soon as I can. If I don't get back to you, though, please feel free to tag me again. I always try to be timely with my GH notifications but some may still slip past me. 😄

**Comment by lyallstewart** [comment]
Perfect, thanks

**Comment by ethan-master-coding** [comment]
@Dusch4593 So should we use the md equivalent of the details and summary tag.

**Comment by Dusch4593** [comment]
> @Dusch4593 So should we use the md equivalent of the details and summary tag.

You could do that! 😄 I only suggested reverting back to the "imgur" PNG link but this might work better:

```
# Heading 1
## Heading 2
### Heading 3
#### Heading 4
##### Heading 5
###### Heading 6
```

**Comment by ethan-master-coding** [comment]
@Dusch4593 I think we should not use a heading I will quote me and you when I changed the HTML heading entry
Me:
> Replaced the image with the headings with actual headings so people can see how big it actually is.

You:

> This is super important considering the next section talks about accessibility. Not only will humans be able to "see" the headers, but so will screen readers hopefully.

**Comment by ethan-master-coding** [comment]
@Dusch4593 For some reason I can't find what it is. and because we are keeping it auto open is there a lot of a point in making it colapible.

**Comment by Dusch4593** [comment]
> @Dusch4593 I think we should not use a heading I will quote me and you when I changed the HTML heading entry Me:
> 
> > Replaced the image with the headings with actual headings so people can see how big it actually is.
> 
> You:
> 
> > This is super important considering the next section talks about accessibility. Not only will humans be able to "see" the headers, but so will screen readers hopefully.

Right, I remember. We should replace `<details>` + `<summary>` with regular markdown headers so they will (a) render as the correct size, and (b) get caught and read by screen readers (to the a11y point).

**Comment by ethan-master-coding** [comment]
@Dusch4593 Correct what about

> @Dusch4593 For some reason I can't find what it is. and because we are keeping it auto open is there a lot of a point in making it colapible.

**Comment by Dusch4593** [comment]
> @Dusch4593 Correct what about
> 
> > @Dusch4593 For some reason I can't find what it is. and because we are keeping it auto open is there a lot of a point in making it colapible.

We want to ensure that when the headings load, they are rendered correctly and are the correct size, and avoid the current issue with [HTML: Headings](https://www.codecademy.com/resources/docs/html/headings

Screen Shot 2022-04-19 at 6 10 25 PM
)

**Comment by ethan-master-coding** [comment]
@Dusch4593 I'm saying why don't we remove them entirely to negate the problem they are automatically open so not much would change anyways.

**Comment by Dusch4593** [comment]
> @Dusch4593 I'm saying why don't we remove them entirely to negate the problem they are automatically open so not much would change anyways.

You mean no image or markup? Just remove it?

**Comment by ethan-master-coding** [comment]
@Dusch4593 Just remove the detail tag so the heading are not collapsible.

**Comment by Dusch4593** [comment]
> @Dusch4593 Just remove the detail tag so the heading are not collapsible.

I don't believe that it would be possible to remove the `<details>` tag and keep just the `[<summary>](https://www.codecademy.com/resources/docs/html/semantic-html/summary)` tag since both are designed to be used together. My current suggestion still stands as I believe that writing actual markdown (instead of using an image or markup) would be the most optimal change to make.

**Comment by ethan-master-coding** [comment]
@Dusch4593 No I meant remove both. And I couldn't fine the md for the details and summary tags

**Comment by Dusch4593** [comment]
> And I couldn't fine the md for the details and summary tags

@ethan-master-coding I'm sorry, what do we mean? I would just use hashtags (`#`) and create regular markdown headers (which will hopefully render in the right size lol 🤞🏻):

```
# Heading 1
## Heading 2
### Heading 3
#### Heading 4
##### Heading 5
###### Heading 6
```

**Comment by ethan-master-coding** [comment]
@Dusch4593 yea

**Comment by ethan-master-coding** [comment]
Guess I pressed the wrong button

**Comment by Dusch4593** [comment]
> Guess I pressed the wrong button

No worries. It happens 😄

**Comment by ethan-master-coding** [comment]
We would just have 

# Heading 1
## Heading 2 

and so on

**Comment by Dusch4593** [comment]
> We would just have
> 
> # Heading 1
> ## Heading 2
> and so on

Exactly

**Comment by ethan-master-coding** [comment]
@Dusch4593 made the change

**Comment by ethan-master-coding** [comment]
Should i make the change to the HTML entry also?

**Review by Dusch4593 — APPROVED** [review]
Looks good for merge! Thanks, @ethan-master-coding! 😄

**Comment by ethan-master-coding** [comment]
Tried adding `<!-- prettier-ignore -->` because the heading normally should have lines separating them.

**Comment by Dusch4593** [comment]
@ethan-master-coding When the changes go live they can be found here: https://www.codecademy.com/resources/docs/markdown/headings

**Comment by ethan-master-coding** [comment]
@Dusch4593 ok. can you quickly review #709 just adding a tag

**Pull request merged**